src/command-line/index.js

```diff
@@ -73,9 +78,9 @@ if (process.argv[1].endsWith(`${require("path").sep}lounge`)) {
 	process.argv[1] = "thelounge";
 }
 
-// `parse` expects to be passed `process.argv`, but we need to remove to give it
-// a version of `argv` that does not contain options already parsed by
-// `parseOptions` above.
+// `parse` expects to be passed `process.argv`, but we need to give it a version
+// of `argv` that does not contain options already parsed by `parseOptions`
+// above.
```

Comment:
This is just fixing my shitty English (rather, write a sentence, re-write it, re-write it again, etc.).